package.json

```diff
@@ -63,9 +63,6 @@
     "packages": [
       "packages/*",
       "packages/docs-website/*"
-    ],
-    "nohoist": [
-      "**/ethers"
```

Comment:
I had to remove this in order to prevent an error in `yarn`. 

https://github.com/yarnpkg/yarn/issues/7807

I didn't see any obvious problems arising from removing these lines...

Reply:
Rolled back in 9430c05 ; the problem stems from having a different version of devtools specified at the top level package.json (not ideal to have to manually change this...)